pages/api/locker/[...lockerId].ts

```diff
@@ -8,20 +8,19 @@ export default async function handler(
 ) {
   if (req.method === 'GET') {
     try {
-      let { lockerId } = req.query
-      let id = lockerId?.toString() || ''
+      const { lockerId } = req.query
+      const id = lockerId?.toString() || ''
```

Comment:
this will still return true even if `id=""` you can set it to null o check if with `id.length`
```suggestion
      const id = lockerId?.toString() || ''
```